src/libpmem2/pmem2_windows.c

```diff
@@ -35,52 +35,60 @@
  */
 
 #include "libpmem2.h"
+#include "out.h"
 #include "pmem2.h"
 #include "pmem2_utils.h"
 
+/*
+ * mapping -- creates or opens a named or unnamed file mapping object
+ *	for a specified file
+ */
+HANDLE
+mapping(const struct pmem2_config *cfg, DWORD protect, int *err)
+{
+	*err = 0;
+	HANDLE fh = CreateFileMapping(cfg->handle,
+		NULL, /* security attributes */
+		PAGE_READWRITE,
+		(DWORD)(0),
+		(DWORD)(0 & 0xFFFFFFFF),
+		NULL);
+
+	if (!(fh) || fh == INVALID_HANDLE_VALUE) {
```

Comment:
According to [the manpage](https://docs.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-createfilemappinga) on error NULL is returned. There is no option to get INVALID_HANDLE_VALUE.